test/functional/cmdLineTests/xlogTests/xlog.xml

```diff
@@ -0,0 +1,251 @@
+		<command>cat gclog</command>
+		<output regex="no" type="success">verbosegc</output>
+	</test>
+
+	<exec command="rm -f gclog" />
+
+	<test id="-Xlog:gc:file=gclog">
+		<command>$EXE$ $ENABLEXLOG$ -Xlog:gc:file=gclog -version</command>
+		<output regex="no" type="success">openjdk version</output>
+		<output regex="no" type="failure">verbosegc</output>
+		<output regex="no" type="failure">JVMJ9VM085</output>
+		<output regex="no" type="failure">JVMJ9VM007</output>
+	</test>
+
+	<test id="-Xlog:gc:file=gclog check file">
+		<command>cat gclog</command>
+		<output regex="no" type="success">verbosegc</output>
+	</test>
+
+	<exec command="rm -f gclog" />
+
+	<test id="-Xlog:gc=off">
+		<command>$EXE$ $ENABLEXLOG$ -Xlog:gc=off -version</command>
+		<output regex="no" type="success">openjdk version</output>
+		<output regex="no" type="required">JVMJ9VM007W</output>
+		<output regex="no" type="failure">verbosegc</output>
+		<output regex="no" type="failure">JVMJ9VM085</output>
+	</test>
+
+	<test id="-Xlog:any+gc=off">
+		<command>$EXE$ $ENABLEXLOG$ -Xlog:any+gc=off -version</command>
+		<output regex="no" type="success">openjdk version</output>
+		<output regex="no" type="required">JVMJ9VM007W</output>
+		<output regex="no" type="failure">verbosegc</output>
+		<output regex="no" type="failure">JVMJ9VM085</output>
+	</test>
+
+	<test id="-Xlog:gc+any=off">
+		<command>$EXE$ $ENABLEXLOG$ -Xlog:gc+any=off -version</command>
+		<output regex="no" type="success">openjdk version</output>
+		<output regex="no" type="required">JVMJ9VM007W</output>
+		<output regex="no" type="failure">verbosegc</output>
+		<output regex="no" type="failure">JVMJ9VM085</output>
+	</test>
+
+	<test id="-Xlog:gc=off,any">
+		<command>$EXE$ $ENABLEXLOG$ -Xlog:gc=off,any -version</command>
+		<output regex="no" type="success">openjdk version</output>
+		<output regex="no" type="required">JVMJ9VM007W</output>
+		<output regex="no" type="failure">verbosegc</output>
+		<output regex="no" type="failure">JVMJ9VM085</output>
+	</test>
+
+	<test id="-Xlog:any,gc=off">
+		<command>$EXE$ $ENABLEXLOG$ -Xlog:any,gc=off -version</command>
+		<output regex="no" type="success">openjdk version</output>
+		<output regex="no" type="required">JVMJ9VM007W</output>
+		<output regex="no" type="failure">verbosegc</output>
+		<output regex="no" type="failure">JVMJ9VM085</output>
+	</test>
+
+	<test id="-Xlog:disable">
+		<command>$EXE$ $ENABLEXLOG$ -Xlog:disable -version</command>
+		<output regex="no" type="success">openjdk version</output>
+		<output regex="no" type="required">JVMJ9VM007W</output>
+		<output regex="no" type="failure">verbosegc</output>
+		<output regex="no" type="failure">JVMJ9VM085</output>
+	</test>
+
+	<test id="-Xlog:disable:">
+		<command>$EXE$ $ENABLEXLOG$ -Xlog:disable: -version</command>
+		<output regex="no" type="success">openjdk version</output>
+		<output regex="no" type="success">JVMJ9VM085E</output>
+		<output regex="no" type="failure">openjdk version</output>
```

Comment:
I don't understand this: how can 'openjdk version' be both a success and a failure condition?

Reply:
It's not, it is a mistake. :)

Reply:
The way the cmdline tester works, it passes if any success condition occurs, and all required conditions occur, and fails if any failure condition occurs. Since "openjdk version" doesn't occur in this test, the mistake didn't cause any problem when running it.

Reply:
Pushed the fix.